Readme.md

```diff
@@ -1,18 +1,83 @@
-<!-- default file list -->
-*Files to look at*:
+<!-- default badges list -->
+[![](https://img.shields.io/badge/Open_in_DevExpress_Support_Center-FF7200?style=flat-square&logo=DevExpress&logoColor=white)](https://supportcenter.devexpress.com/ticket/details/T501713)
+[![](https://img.shields.io/badge/📖_How_to_use_DevExpress_Examples-e9f6fc?style=flat-square)](https://docs.devexpress.com/GeneralInformation/403183)
+<!-- default badges end -->
+# Popup Control for ASP.NET Web Forms - How to use one pop-up across all pages of an application
+
+You can use one [Popup Control](https://docs.devexpress.com/AspNet/3582/components/docking-and-popups/popup-control) across all pages of your application to improve performance. This example demonstrates how to create a popup control on the master page and invoke it from multiple pages. The pop-up window's content depends on the page that shows this window.
+
+![Popup Window](popup.png)
+
+In this example, the **WebForm1** page populates the pop-up window with content during control initialization. The **WebForm2** page specifies the window's content once the control sends a callback to the server. Use **Init approach** and **Callback approach** menu items to switch between these pages. Click the **Show Popup** button to show the pop-up window.
```

Comment:
We get used to phrases like "the Height property" or "the Button class", but sometimes it's more natural to mention the name after the noun. I'm not sure why exactly it feels more natural to switch the order in this case. Maybe it's because of the numbers in the names of pages. In the third sentence, you need to read through two long names before you understand what the sentence talks about (menu items). 

 ```suggestion
In this example, the page **WebForm1** populates the pop-up window with content during control initialization. The page **WebForm2** specifies the window's content once the control sends a callback to the server. Use menu items **Init approach** and **Callback approach** to switch between these pages. Click the **Show Popup** button to show the pop-up window.
```